### PR TITLE
test(e2e): FIFO SNS and SQS

### PR DESCRIPTION
**Pull request by paragbhingre**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: e2e/worker/worker_test.go [hunk 1]
```diff
@@ -78,6 +78,12 @@ var _ = Describe("Worker Service E2E Test", func() {
 publish:
   topics:
   - name: events
+  - name: mytopic
```
Comment (iamhopaul123) [inline e2e/worker/worker_test.go:81]: does the e2e test work with this duplication...
```suggestion
```

**Review by iamhopaul123 — COMMENTED** [review]
How do we make sure FIFO really work (like the messages are passed through at strict FIFO pattern) without updating the service code 🤔

File: e2e/worker/worker_test.go [hunk 2]
```diff
@@ -160,14 +166,43 @@ publish:
...
-				TopicSubscriptions: []string{fmt.Sprintf("%s:%s", workerServiceName, "processed-msg-count")},
+				Name:            counterServiceName,
+				SvcType:         "Worker Service",
+				Dockerfile:      "./counter/Dockerfile",
+				NoSubscriptions: true,
 			})
 			Expect(err).NotTo(HaveOccurred())
 		})
 
+		It("adds a topic and queues to push to the service", func() {
```
Comment (Lou1415926) [inline e2e/worker/worker_test.go:177]: An alternative would be to write add a `manifest.yml` file under `copilot/worker/manifest.yml`, like what we do with `multi-svc-app`.

**Review by Lou1415926 — COMMENTED** [review]
same question as PH's: do we want to test that the messages are indeed FIFO?

**Comment by Lou1415926** [comment]
Closing this for now. We can reopen it in the future if it's deemed necessary to include FIFO in e2e tests.

**Pull request closed without merging**